plugins/generator-1.17.1/forge-1.17.1/biome.definition.yaml

```diff
@@ -0,0 +1,18 @@
+templates:
+  - template: biome.java.ftl
+    name: "@SRCROOT/@BASEPACKAGEPATH/world/biome/@NAMEBiome.java"
+  - template: tree_fruits.java.ftl
+    name: "@SRCROOT/@BASEPACKAGEPATH/world/level/levelgen/feature/treedecorators/@NAMEDecorator.java"
+    condition: hasFruits()

```

Comment:
hasFruits -> haveFruits

Reply:
Does biome have fruits

but this also works

Has biome fruits?

So I think this is ok

Reply:
Fruits are on custom fruits, so I don’t think it’s ok.